app/grandchallenge/components/models.py

```diff
@@ -1477,7 +1477,9 @@ def validate_user_upload(self, user_upload):
                 try:
                     value = json.loads(user_upload.read_object())
                 except JSONDecodeError as error:
-                    raise ValidationError(error)
+                    raise ValidationError(
+                        f"The file is not valid JSON. {error}"
+                    )
```

Comment:
```suggestion
                    ) from error
```

Reply:
We should be doing this everywhere this pattern occurs, so curious if this works here.